apps/web/src/pages/templates/components/TemplatePushEditor.tsx

```diff
@@ -34,16 +25,16 @@ export function TemplatePushEditor({
   return (
     <>
       {!hasActiveIntegration ? <LackIntegrationAlert channelType={ChannelTypeEnum.PUSH} /> : null}
-      <StepSettings index={index} />
+      <StepSettings />
       <Controller
-        name={`${path}.title` as any}
+        name={`${stepFormPath}.template.title` as any}
```

Comment:
in many places I did changes like this - use dynamic path to the form step/variant